pkg/build/apis/build/types.go

```diff
@@ -540,6 +543,7 @@ const (
 	StatusMessageNoBuildContainerStatus          = "The pod for this build has no container statuses indicating success or failure."
 	StatusMessageFailedContainer                 = "The pod for this build has at least one container with a non-zero exit status."
 	StatusMessageGenericBuildFailed              = "Generic Build failure - check logs for details."
+	StatusMessageOOMKilled                       = "The build pod was killed for 'Out of memory'"
```

Comment:
"The build pod was killed due to an out of memory condition"  (we don't actually know if it's the node that ran out of memory or the container exceeded its cgroup memory limit, right?  @derekwaynecarr @mfojtik @sjenning)